sys/include/test_utils/netdev_ieee802154_minimal.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2022 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    test_utils_netdev_ieee802154_minimal Minimal netdev IEEE 802.15.4 device processing
+ * @ingroup     sys
+ *
+ * @{
+ * @file
+ * @brief       Provides basic functionalities to interact with an
+ *              IEEE 802.15.4 networking device which implements the
+ *              @ref drivers_netdev_api.
+ *
+ * To use the functionalities, include the module
+ * `USEMODULE += test_utils_netdev_ieee802154_minimal`.
+ * The test application should provide:
+ *  - device initialization, via the implementation of @ref netdev_ieee802154_minimal_init_devs
+ *  - number of devices to test, via the definition of @ref NETDEV_IEEE802154_MINIMAL_NUMOF in `init_dev.h`
```

Comment:
```suggestion
 *  - number of devices to test, via the definition of @ref NETDEV_IEEE802154_MINIMAL_NUMOF
 *    in `init_dev.h`
```

The CI frowns very hard on this > 100 char line (: